.grit/patterns/js/codecept_to_playwright.md

```diff
@@ -695,74 +696,8 @@ let myData = [
 ];
 
 for (const current of myData) {
-  test(`Trivial test @Projects @Studio @Email`, async ({ page, factory, context }) => {
+  test(`Trivial test @Email @Studio @Projects`, async ({ page, factory, context }) => {
     console.log(current.capital);
   });
 }
-```
-
-## Intelligently converts stringlike locators
```

Comment:
Why are we modifying and removing tests?

Reply:
Oops, deleted test still passes, adding back, modifications are just to the ordering of the tags

Reply:
Do the tests fail in the original order?

Reply:
yes because the order in which we visit the tags changes, new version preserves the tag ordering between codecept and playwright